kolibri/core/assets/src/core_app_instance.js

```diff
@@ -3,6 +3,9 @@
 require('normalize.css');
 require('./font-NotoSans.css');
 require('./core-global.styl');
+require('webpack-material-design-icons');
```

Comment:
The use of this here is what makes me wonder if we might prefer material design buttons atop?